Add configure-vscode-for-bazel to generate launch.json in VS Code (#5)
luminartech/dev-tools: README.md

<!-- mdformat-toc end -->

## Tools

### Configure VS Code for Bazel

> If you want to debug C++ Bazel targets in VS Code, you can use the `configure-vscode-for-bazel` to generate a `.vscode/launch.json` file.

Reviewer: ```suggestion
If you want to debug C++ Bazel targets in VS Code, you can use `configure-vscode-for-bazel` to generate a `.vscode/launch.json` file.
```

> This file then contains targets for debugging in VS Code.
> To generate a set of targets, run the following command:

Reviewer: ```suggestion
This file then contains all targets for debugging in VS Code.
To generate a defined set of targets, run
```
